### DISCUSSION TITLE
Allow collaboratively editing commit messages as part of PR review

### DISCUSSION THREAD

**Discussion by mathiasbynens** [community/community#4433] [Pull Requests]
GitHub's PR review interface is excellent. I like that you can make code suggestions that can easily be applied as well.

One thing that's missing is the ability to collaborate on the final commit message that will be used when the PR gets merged. Currently it's possible for the repo owner to change the commit message in the UI right before clicking the merge button, but there is no way (through the UI) to do that ahead of time, let alone to suggest changes to the commit message.

**It would be amazing if the commit message would show up as if it were a separate file in the patch, and could be collaboratively reviewed in similar fashion.**

**Comment by ricera** [comment]
Yeah, similar to gerrit.
